pkgs/development/python-modules/htmltreediff/default.nix

```diff
@@ -21,6 +21,6 @@ buildPythonPackage rec {
     description = " Structure-aware diff for html and xml documents";
     homepage = "https://github.com/christian-oudard/htmltreediff";
     license = licenses.bsdOriginal;
-    maintainers = with maintainers; [ ma27 ];
+    maintainers = with maintainers; [ ];
```

Comment:
This project did not receive any commits for 10 years. I think this can be potentially removed.

Reply:
for a python package, that is especially bad

Reply:
What's the guideline on removing packages? What about projects in compiled languages written in C last updated in 2004, for instance?

Reply:
@siraben if it's still working and providing utility to someone, IMO we should keep it around "forever". If something hasn't broken in 16 years, which would it break now?